docs/spelling_wordlist.txt

```diff
@@ -18,9 +18,18 @@ Dramatiq
 enqueued
 formatters
 Godwin
+healthchecks
 linting
+localhost
 middleware
+namespaces
+programmatically
+pum
+pycharm
+queueing
 quickstart
 reimplement
+schemas
 sync
+tox
 Travis
```

Comment:
Ha, you managed to get the spell check working on your host ? Nice job !

Reply:
Yep, I needed the aspell-en Arch package.